### PR TITLE
[Add function] add 'Split Space and ISO and delete RGUI for Promise 87

### PR DESCRIPTION
**Pull request by spbgzh**
## Description
add 'Split Space and ISO for Promise 87

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [x] New feature
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR
https://github.com/qmk/qmk_firmware/pull/17525

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by fauxpark — APPROVED** [review]

File: keyboards/wuque/promise87/ansi/keymaps/default_tkl_f13_ansi_iso/keymap.c [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] ={
+    [0] = LAYOUT_tkl_f13_ansi_iso(
+        KC_ESC,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_F13,  KC_PSCR, KC_SLCK, KC_PAUS,
```
Comment (fauxpark) [inline keyboards/wuque/promise87/ansi/keymaps/default_tkl_f13_ansi_iso/keymap.c:21]:
```suggestion
        KC_ESC,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_F13,  KC_PSCR, KC_SCRL, KC_PAUS,
```

File: keyboards/wuque/promise87/ansi/keymaps/default_tkl_f13_ansi_split_space/keymap.c [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] ={
+    [0] = LAYOUT_tkl_f13_ansi_split_space(
+        KC_ESC,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_F13,  KC_PSCR, KC_SLCK, KC_PAUS,
```
Comment (fauxpark) [inline keyboards/wuque/promise87/ansi/keymaps/default_tkl_f13_ansi_split_space/keymap.c:21]:
```suggestion
        KC_ESC,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_F13,  KC_PSCR, KC_SCRL, KC_PAUS,
```

File: keyboards/wuque/promise87/wkl/keymaps/default_tkl_f13_ansi_wkl_iso/keymap.c [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] ={
+    [0] = LAYOUT_tkl_f13_ansi_wkl_iso(
+        KC_ESC,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_F13,     KC_PSCR, KC_SLCK, KC_PAUS,
```
Comment (fauxpark) [inline keyboards/wuque/promise87/wkl/keymaps/default_tkl_f13_ansi_wkl_iso/keymap.c:21]:
```suggestion
        KC_ESC,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_F13,     KC_PSCR, KC_SCRL, KC_PAUS,
```

File: keyboards/wuque/promise87/wkl/keymaps/default_tkl_f13_ansi_wkl_split_space/keymap.c [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] ={
+    [0] = LAYOUT_tkl_f13_ansi_wkl_split_space(
+        KC_ESC,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_F13,     KC_PSCR, KC_SLCK, KC_PAUS,
```
Comment (fauxpark) [inline keyboards/wuque/promise87/wkl/keymaps/default_tkl_f13_ansi_wkl_split_space/keymap.c:21]:
```suggestion
        KC_ESC,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_F13,     KC_PSCR, KC_SCRL, KC_PAUS,
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by spbgzh** [comment]
Emmm I just request review, I dont know why it removed

**Review by fauxpark — APPROVED** [review]

**Review by noroadsleft — COMMENTED** [review]
Is the `wuque/promise87/ansi` version just short for `winkey` (as opposed to `wkl` winkeyless)?

**Comment by spbgzh** [comment]
> Is the `wuque/promise87/ansi` version just short for `winkey` (as opposed to `wkl` winkeyless)?

wkl is for win key less

**Comment by noroadsleft** [comment]
> just add the win key from wkl version

Okay, in my opinion that should be renamed during the next Breaking Changes cycle (2023q1).

I have some change requests for you but I'm on mobile right now; will have to wait until I'm back at my PC.

**Comment by spbgzh** [comment]
@noroadsleft ok👌

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**